### DISCUSSION TITLE
Not beeing able to qualify for teacher benefits because GitHub

### DISCUSSION THREAD

**Discussion by jzscsaba** [community/community#51408] [GitHub Education]
Hi,

I was not able to qualify for teacher benefits because the following error message appeared:
"We require applicants of John von Neumann University to use one of these school-issued email addresses to apply:
**hallgato.uni-neumann.hu**"
However, the subdomain "**hallgato**" (the word means **student** in Hungarian) is for students only. The teacher accounts at John von Neumann university are either in a subdomain corresponding to the faculty (e.g. [...@gamf.uni-neumann.hu](mailto:...@gamf.uni-neumann.hu)) or without the subdomain indicating the faculty (e.g. @uni-neumann.hu) or in case of the new e-mail accounts are of form [...@nje.hu](mailto:...@nje.hu).
I submitted a ticket regarding this issue, but it was simply closed.
Any ideas?

**Comment by F0XN0T** [comment]
I understand your frustration with the difficulties you're facing in accessing the GitHub Student Developer Pack. It's concerning to hear that despite creating multiple tickets and providing personal information, you haven't received a satisfactory response from GitHub. It seems like this issue is affecting not just you, but many other students and creators as well.

It's important to note that personal information should always be handled with care and privacy should be respected. However, it's also crucial for platforms like GitHub to provide timely and humane support to their users.

If you're still facing issues, one option could be to try reaching out to GitHub's customer support team again and explaining the situation in detail. It may also be helpful to connect with other affected students and creators to see if there are any collective actions that can be taken.

I hope this issue can be resolved soon, as it's affecting a large number of users and is urgent in nature.
